R/trainSLOPE.R

```diff
@@ -54,7 +75,7 @@ trainSLOPE <- function(x,
                        measure = c("mse",
                                    "mae",
                                    "deviance",
-                                   "missclass",
+                                   "misclass",
```

Comment:
I love to correct typos, but this creates problems with backwards compatibility. Can you please include both `misclass` and `missclass` and deprecate `missclass`, i.e. issue a warning if that word is uses (but keep the function from failing)? The `missclass` option can be removed from the package after another version update.

Reply:
Solution: https://github.com/BioGenies/SLOPE/blob/master/R/trainSLOPE.R#L83

A warning is raised but `missclass` is automatically replaced to `misclass`.